tutorials/scripting/gdscript/gdscript_exports.rst

```diff
@@ -9,11 +9,11 @@ Introduction to exports
 In Godot, class members can be exported. This means their value gets saved along
 with the resource (such as the :ref:`scene <class_PackedScene>`) they're
 attached to. They will also be available for editing in the property editor.
-Exporting is done by using the ``@export`` annotation::
+Exporting is done by using the ``@export`` annotation.
 
-    extends Button
+::
 
-    @export var number = 5
+    @export var number: int = 5
 
 In that example the value `5` will be saved and visible in the property editor.
```

Comment:
This probably needs two ticks on each side, I think it should be formatted as code.